coveralls/api.py

```diff
@@ -276,9 +317,14 @@ def submit_report(self, json_string):
                 new_id = '{}-{}'.format(
                     self.config.get('service_job_id', 42),
                     random.randint(0, sys.maxsize))
-            print('resubmitting with id {}'.format(new_id))
-
             self.config['service_job_id'] = new_id
+            log.warning(
```

Comment:
It seems to me that with the fixes in coveralls.io, it is no longer necessary to resubmit, nor would it help to do that with a different service_job_id if the first submission already now sets a service_job_id.